terraform/modules/data-share-service/ec2.tf

```diff
@@ -45,7 +45,7 @@ resource "aws_security_group" "rds_bastion_host_sg" {
     protocol    = "tcp"
     from_port   = 22
     to_port     = 22
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = var.externally_allowed_cidrs
```

Comment:
Unfortunately, this doesn't mean we can remove the tfsec ingress rule, but it's definitely better

Reply:
Awh why not?

Reply:
I think because the IPs are still "public internet IPs" it's not a fan